src/xsar/radarsat2_meta.py

```diff
@@ -312,11 +337,16 @@ def flip_line_da(self):
         system where the range increases to the right and the zero-Doppler time increases downward. Note that this is
         not necessarily the top-left pixel of the image block in the final product.`
         """
-        pass_direction = self.dt.attrs['passDirection']
-        samples_depending_ds = ['geolocationGrid']
-        if pass_direction == 'Ascending':
+        pass_direction = self.dt.attrs["passDirection"]
+        samples_depending_ds = ["geolocationGrid"]
+        if pass_direction == "Ascending":
             for ds_name in samples_depending_ds:
-                self.dt[ds_name] = self.dt[ds_name].copy().isel(line=slice(None, None, -1))\
-                    .assign_coords(line=self.dt[ds_name].ds.line)
+                ds = self.dt[ds_name].ds
```

Comment:
Are you sure about the `self.dt[ds_name] = ds` in here ? Because you got access to it using `ds = self.dt[ds_name].ds`

Reply:
Unless this point (which appears also on other meta classes), this is good for me.

Reply:
yes sure this syntax is working. I've checked before after line 344. The data is correctly flipped.

Reply:
I mark as revolved so you can merge if it's ok.